test/Passes/xsmm-combine.mlir

```diff
@@ -0,0 +1,247 @@
+//RUN: tpp-opt -verify-xsmm-calls  --combine-xsmm-op-optimization  -verify-xsmm-calls %s --split-input-file | FileCheck %s
```

Comment:
General comments on the tests: They are really difficult to follow. Can we simplify them?

Reply:
We're getting the IR from mlir-gen, hence simplifying the IR isn't trivial.